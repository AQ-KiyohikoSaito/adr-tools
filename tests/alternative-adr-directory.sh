export ADR_DIR=alternative-dir
adr new Example ADR
ls -R .
