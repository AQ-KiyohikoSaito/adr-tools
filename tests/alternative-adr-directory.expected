+ ADR_DIR=alternative-dir
+ export ADR_DIR
+ adr new Example ADR
alternative-dir/0001-example-adr.md
+ ls .
alternative-dir
+ ls alternative-dir
0001-example-adr.md
